src/python_testing/TC_SEAR_1_5.py

```diff
@@ -117,24 +117,28 @@ async def test_TC_SEAR_1_5(self):
             test_step = "Manually intervene to put the device in a state that prevents it from executing the SkipArea command \
                   (e.g. set CurrentArea to null or make it not operate, i.e. be in the idle state)"
```

Comment:
```suggestion
                  (e.g. set CurrentArea to null or make it not operate, i.e. be in the idle state). Ensure that SelectedArea is not empty."
```

Reply:
It looks like the current text is pulled straight from the TP, so adjusting this should also mean we should ideally tweak the TP? and if that is added , should there be a read to assert this?